src/main/java/ch/njol/skript/expressions/ExprQueueStartEnd.java

```diff
@@ -0,0 +1,118 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.registrations.Feature;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.util.SkriptQueue;
+
+import java.util.Arrays;
+
+@Name("Queue Start/End (Experimental)")
+@Description("""
+	Requires the <code>using queues</code> experimental feature flag to be enabled.
+	
+	The first or last element in a queue. Asking for this does <b>not</b> remove the element from the queue.
+	
+	This is designed for use with the <code>add</code> changer: to add or remove elements from the start or the end of the queue.
+	""")
+@Examples({
+	"""
+		set {queue} to a new queue
+		add "hello" to {queue}
+		add "foo" to the start of {queue}
+		broadcast the first element of {queue} # foo
+		broadcast the first element of {queue} # hello
+		# queue is now empty"""
+})
```

Comment:
```suggestion
@Examples("""
		set {queue} to a new queue
		add "hello" to {queue}
		add "foo" to the start of {queue}
		broadcast the first element of {queue} # foo
		broadcast the first element of {queue} # hello
		# queue is now empty
		""")
```
Formatting? To make it consistent with the annotation above.